radixdlt-java/src/main/java/com/radixdlt/client/application/translate/AtomToActionsMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.radixdlt.client.application.translate;
+
+import com.radixdlt.client.application.identity.RadixIdentity;
+import com.radixdlt.client.core.atoms.Atom;
+import io.reactivex.Observable;
+
+public interface AtomToActionsMapper<T> {
```

Comment:
The name would suggest (to me anyway) that the implementations return an `Action`. But for example `AtomToTokenTransferActionsMapper` returns `TokenTransfer` rather than a `TransferTokensAction`. Similarly `AtomToMessageActionsMapper` returns `UnencryptedData` instead of `SendMessageAction`. So at the least, we might want to consider renaming it.

Reply:
Also, should we have a common base class / interface / something to all `Action`s?

Reply:
I think even having it just be a marker interface has merit. We should add it.